src/Elliptic/Systems/SelfForce/Scalar/BoundaryConditions/Factory.hpp

```diff
@@ -0,0 +1,13 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "Elliptic/Systems/SelfForce/Scalar/BoundaryConditions/Angular.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace ScalarSelfForce::BoundaryConditions {
```

Comment:
dox for the namespace? Sorry if it is somewhere else and I didn't see it. Not sure where the best place is either.